javascript/javascript1/week3/cactusIO.js

```diff
@@ -0,0 +1,42 @@
+console.log(`======cactusIO=====`);
+const durationLimit = 120;
+let activities = [];
+
+function addActivity(date, activity, duration) {
+    const newActivites = { date: date, activity: activity, duration: duration };
+    activities.push(newActivites)
+}
+
+function showStatus() {
+    const numbersOfActiveties = activities.length;
+    if (numbersOfActiveties === 0) {
+        console.log("Add some activities before calling showStatus");
+        return;
```

Comment:
If you want this in one line you can do:
`return console.log("Add some activities ...`
It doesn't actually return anything, but it console logs and returns.